content/solutions/bitbucketserver.adoc

```diff
@@ -0,0 +1,20 @@
+---
+layout: solution
+title: "Bitbucket Server"
+---
+
+link:https://www.atlassian.com/software/bitbucket/enterprise/data-center[Bitbucket Server] is a Git repository management solution designed for professional teams. It’s part of the link:http://www.atlassian.com/[Atlassian] product family along with Jira, Confluence, and many more tools designed to help teams unleash their full potential.
+
+To integrate it with Jenkins, you can install the link:https://plugins.jenkins.io/atlassian-bitbucket-server-integration[Bitbucket Server integration for Jenkins] plugin. This plugin, which is built and supported by Atlassian, is the easiest way to connect these two tools. It features:
```

Comment:
You can use the plugin macro here as well for the plugins.jenkins.io url

Reply:
Thanks @timja . I've updated it so it uses the plugin macro.